operations/messaging-and-math/developer-relations.md

```diff
@@ -14,6 +14,14 @@ Some activities that fall under formal Developer Relations:
 
 Please reach out to the [Community Team](mailto:community@mattermost.com) if you have any questions or would like assistance for any activities in their milieu.
 
+## Definitions
+Event Engaged Attendee
```

Comment:
```suggestion
* Event Engaged Attendee
```

Reply:
@jasonblais - yes - with the conference attended by about 50k people, my talk was an average size (2500) for the stage/track and topic